app/helpers/form_helper.rb

```diff
@@ -0,0 +1,32 @@
+module FormHelper
+  def form_errors(errors)
+    tag.ul(class: %w[help-block error-block]) do
+      safe_join(errors.map { |e| tag.li(e) })
+    end
+  end
+
+  def form_group(form, field_name, has_label: true, label: nil, help: nil, attributes: {}, &block)
```

Comment:
Let's drop the has_label field. We should expect that if users have a case that doesn't need a label that they can then render their form without the helper.

Reply:
removed `has_label`.  Thanks.